homeassistant/components/nest/legacy/local_auth.py

```diff
@@ -4,10 +4,10 @@
 
 from nest.nest import AUTHORIZE_URL, AuthorizationError, NestAuth
 
+from homeassistant.components.nest import config_flow
```

Comment:
Can we change to something like this?
```py
from ..config_flow import register_flow_implementation, ...
```

Reply:
Thanks, I was struggling with the syntax.